lib/terraforming/resource/ec2.rb

```diff
@@ -73,7 +73,7 @@ def block_device_ids_of(instance)
 
       def block_devices_of(instance)
         return [] unless instance.block_device_mappings.length > 0
-        @client.describe_volumes(volume_ids: block_device_ids_of(instance)).volumes
+        @client.describe_volumes(volume_ids: block_device_ids_of(instance)).map(&:volumes).flatten
```

Comment:
__[rubocop]__
* Line is too long. [98/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)